content/en/docs/reference/scheduling/config.md

```diff
@@ -285,7 +260,13 @@ only has one pending pods queue.
 * A plugin enabled in a v1beta2 configuration file takes precedence over the default configuration for that plugin.
 
 * Invalid `host` or `port` configured for scheduler healthz and metrics bind address will cause validation failure.
+{{% /tab %}}
 
+{{% tab name="v1beta2 → v1beta3" %}}
+* Three plugins' weight are increased by default:
+  * `InterPodAffinity` from 1 to 2
```

Comment:
Found this PR while trying to make changes to my [PR](https://github.com/kubernetes/website/pull/30441/). Thank you for doing this @kerthcet. The weight changes look correct to me. I'll leave the migration doc to @damemi and @alculquicondor